mythril/laser/ethereum/transaction/symbolic.py

```diff
@@ -73,7 +73,7 @@ def execute_contract_creation(
     del laser_evm.open_states[:]
 
     new_account = laser_evm.world_state.create_account(
-        0, concrete_storage=True, dynamic_loader=None, creator=CREATOR_ADDRESS
+        0, concrete_storage=False, dynamic_loader=None, creator=CREATOR_ADDRESS
```

Comment:
Why make the storage symbolic?

Reply:
That flag doesn't seem to make storage symbolic, it just assigns the storage slot to 0 before returning, if the slot doesn't exist. We should do something about it.

Reply:
It does. If concrete_storage is true and the key is not found, then it returns a symbol with value 0. Otherwise it creates a new unconstrained variable which is returned.
This is the difference between concretely initialized storage (all at 0) and symbolic storage right?

Reply:
`Otherwise it creates a new unconstrained variable which is returned` it seems to assign the storage slot to 0 and then return the value.

Reply:
Ah, that's a bug. It should put a new storage symbol there

Reply:
Still, there is no need for this change right?
Once the behaviour would be fixed, then this would break